change-the-middle-management-mindset.md

```diff
@@ -0,0 +1,83 @@
+    - We have innovation days (10% time set aside on calendars, broadcast, you can participate). Choose amongst a few and you get three days to form a new team, work on things together. Hackathon for customer-facing or internal projects (prototypes). Could have an IS day where the metric is: Did you work on someone else's code base? Stand it up and then it becomes available for others to use. Come over to collaborate. Opening an issue=1 point; filing a PR= 5 points; 20 points for merging a PR. Then have an award (two weeks off at Hawaii, all paid with your family; or 3-6 months off to work on your own innovative products; or 6 months part time). Every year could have 1-10 people for IS awards--recognition from the CEO.
+* Fellows program if you can achieve guest TC; you get one day a week to do cross-platform work (they report on it as a measurement of success rather than a measurement of loss). Planning is good as a part of resource allocation (might have to change expectations; we're one team, a whole group).
+* Champions program recruited from any group: could be a Middle-Management champion who has gone through the process before. See who are the new believers. Could put badges on people's names in the directory. Could do cardboard cut-outs of the champion with an IS t-shirt ;-)
+
+* Built-into the whole design of IS process. Delivery teams don't own any code themselves; they change code amongst the product teams. Middle Management for product teams know that requests will come in for their code to be changed. They need to have their developers provide mentorship to the delivery team developers. We have different BLs; each is represented by a delivery team. Product teams focus on larger architectural decisions, ensure that the delivery teams don't mess too much with the Products.
+    - We had globalization teams responsible for countries; had constraints (compliance) for each country. Those teams are always good about IS. But for some reason they went away (restructuring).
+
+* Architectural solutions
+    - Microservices architecture: creates incentive organizationally for people to collaborate. If a bug occurs, then it creates a problem for the users. 
+        - but it might be violating an SLA if the bug isn't being fixed
+        - the problem comes with feature requests or affects design (going beyond bug fixes)
+    - Developing platforms is an ideal IS use case (hackathon to build applications on top)
+    - SW Architects have to have an IS mentality and work together.
+        - newer companies with open source developers have that mindset
+
+* Empowering Middle Management - IS readiness checklist; Middle Management should partner with their developers. What are the opportunities out there. Can we come up with justification for you to spend any time on this (how does this tie together with our KPIs)
```

Comment:
Are you talking about justifying spending time as an IS contributor, IS maintainer, or both?

Reply:
I think it's spending time with the team, as they try IS that the objective is in aligned with MM goals [the things they care about] as the team also needs to welcome the collaboration.